models/Discord/Commands/DiscordCommandPost

```diff
@@ -0,0 +1,21 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandPost extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("post", "post the input as yogbot.", 'post', subsystem);
+  }
+
+  onRun(message, permissions, args) {
+    var config = this.subsystem.manager.getSubsystem("Config").config;
+
+    if(args.length < 1) {
+        message.delete(0);
```

Comment:
Change this to `args.length > 0` and change the delete and return for `message.channel.send(args.join(" "))`
Little bit less code this way.

Reply:
@AshCorr what do you want me to change the delete and return to